vendor-extra/LiberoPatternsBundle/src/ViewConverter/Inline/ItalicVisitor.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\LiberoPatternsBundle\ViewConverter\Inline;
+
+use FluentDOM\DOM\Element;
+use FluentDOM\DOM\Node\ChildNode;
+use Libero\Views\InlineViewConverter;
+use Libero\Views\InlineViewConverterVisitor;
+use Libero\Views\View;
+use function Functional\map;
+
+final class ItalicVisitor implements InlineViewConverterVisitor
+{
+    private $inlineConverter;
+
+    public function __construct(InlineViewConverter $inlineConverter)
+    {
+        $this->inlineConverter = $inlineConverter;
+    }
+
+    public function visit(ChildNode $object, View $view, array &$context = []) : View
+    {
+        if (!$object instanceof Element || 'i' !== $object->nodeName || 'http://libero.pub' !== $object->namespaceURI) {
```

Comment:
Opened https://github.com/ThomasWeinert/FluentDOM/pull/84 to simplify this a little.